### PR TITLE
Fix build error

### PR DESCRIPTION
**Pull request by koertho**
This PR fix some build errors due missing closing tags.

### CODE REVIEW

File: docs/manual/backend-configuration/store-configuration/checkout-flow/payment-methods/payment-method-saferpay/_index.en.md [hunk 1]
```diff
@@ -57,8 +57,9 @@ In the Saferpay login area, you can create and configure payment pages (PP) unde
...
+<table><thead><tr><th>Setting</th> <th>Default setting</th> <th>Description</th> </tr></thead><tbody><tr><td>Name of the configuration</td> <td>-</td> <td> A name of your choice.
+
+This value is specified in the payment provider configuration in Contao in the *Payment page configuration (VTCONFIG)* field.</td> </tr><tr><td>Description</td> <td>-</td> <td> Description that will be displayed to the customer on the PP. Additionally, you can set here whether the PP is active or inactive. </td> </tr><tr><td>Mode selection</td> <td>PayInit</td> <td> Here you can select which attributes have priority: - **PayInit** Attributes in the PayInit have priority over the values specified in the form.
```
Comment (aschempp) [inline docs/manual/backend-configuration/store-configuration/checkout-flow/payment-methods/payment-method-saferpay/_index.en.md:62]: shouldn't we add a closing `{{% /notice %}}` here instead?

File: docs/manual/backend-configuration/store-configuration/general-settings/notifications_overview/_index.en.md [hunk 2]
```diff
@@ -18,7 +18,8 @@ If you imagine `Benachrichtigungszentrum` detached from Isotope eCommerce, some
 
 - A notification always has a trigger, and since that trigger should potentially trigger several different messages, a notification consists of one or more messages.
 
-{{% notice warning %}}Note the different usage of `Benachrichtigung` and a single `Nachricht`. A `Benachrichtigung` is a vessel of one or more `Nachrichten`.{{% /notice %}}
+{{% notice warning %}}Note the different usage of `Benachrichtigung` and a single `Nachricht`. A `Benachrichtigung` is a vessel of one or more `Nachrichten`.
```
Comment (aschempp) [inline docs/manual/backend-configuration/store-configuration/general-settings/notifications_overview/_index.en.md:21]: is this change nescessary?

File: docs/manual/backend-configuration/store-configuration/general-settings/notifications_overview/_index.de.md [hunk 3]
```diff
@@ -16,7 +16,8 @@ Stellst du dir das `Benachrichtigungszentrum` losgelöst von Isotope eCommerce v
 
 * Eine Benachrichtigung hat immer einen Auslöser und da bei diesem Auslöser potenziell mehrere verschiedene Nachrichten ausgelöst werden sollen, besteht eine Benachrichtigung aus einer oder mehreren Nachrichten.
 
-{{% notice warning %}}Merke dir die unterschiedliche Verwendung von <code>Benachrichtigung</code> und einer einzelnen <code>Nachricht</code>. Eine <code>Benachrichtigung</code> ist ein Gefäss von einer oder mehreren <code>Nachrichten</code>.{{% /notice %}}
+{{% notice warning %}}Merke dir die unterschiedliche Verwendung von <code>Benachrichtigung</code> und einer einzelnen <code>Nachricht</code>. Eine <code>Benachrichtigung</code> ist ein Gefäss von einer oder mehreren <code>Nachrichten</code>.
```
Comment (aschempp) [inline docs/manual/backend-configuration/store-configuration/general-settings/notifications_overview/_index.de.md:19]: is this change nescessary?

File: docs/manual/backend-configuration/store-configuration/checkout-flow/payment-methods/payment-method-saferpay/_index.en.md (hunk 1 above)
Comment (koertho) [reply docs/manual/backend-configuration/store-configuration/checkout-flow/payment-methods/payment-method-saferpay/_index.en.md:62]: In the german version there is no notice, so I decided to remove it from here.

File: docs/manual/backend-configuration/store-configuration/general-settings/notifications_overview/_index.en.md (hunk 2 above)
Comment (koertho) [reply docs/manual/backend-configuration/store-configuration/general-settings/notifications_overview/_index.en.md]: I got an error on this page and this fixed it somehow. I reverted it and it still builds :man_beard:

**Comment by aschempp** [comment]
Thank you @koertho

**Pull request merged**